### PR TITLE
Add Description to Module View

### LINKED ISSUE #11: [FEATURE REQUEST] Add description/information on Modules tab
**Is your feature request related to a problem? Please describe.**
The current modules page does not all you to view the information about modules without going to an active agent.

**Describe the solution you'd like**
Add the module description/information to the modules page to match how 'execute module' displays the info.

Modules Tab:
image

Execute Modules:
image

**Describe alternatives you've considered**
The current implementation works, but it would definitely make it easier to use the modules page as a reference section.

**Additional context**
N/A

### PR DESCRIPTION
**Pull request by logicsec-zz**
Fixes #11 

## Type of change
Feature Request

- [X] New feature (non-breaking change which adds functionality)

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [X] vue-cli-service lint

# Checklist:

- [X] My code follows the style guidelines of this project
- [X] I have performed a self-review of my own code
- [X] My changes generate no new warnings
- [X] New and existing unit tests pass locally with my changes
- [X] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Comment by vinnybod** [comment]
I just have a few styling tweeks and then I think this is good to go.

Can we wrap the title for each field in a `<b></b>` tag like we do [here](https://github.com/BC-SECURITY/Starkiller/blob/master/src/components/InfoViewer.vue#L13) and display the comments and author without the brackets by joining them like we do [here](https://github.com/BC-SECURITY/Starkiller/blob/master/src/components/agents/AgentExecuteModule.vue#L152)? `.join(', ')` for author and `.join('\n')` for comments. And then just removing the `<br>` since `<p>` seems to provide enough spacing.

The result:
Screen Shot 2020-04-12 at 6 25 21 PM

**Comment by logicsec-zz** [comment]
Yea I think that works for me :)

**Comment by vinnybod** [comment]
Okay cool, we can merge it to dev once you push those changes up!

**Comment by logicsec-zz** [comment]
Should see it now :)

File: src/views/Modules.vue [hunk 1]
```diff
@@ -11,8 +11,26 @@
...
+          <p><b>Background:</b> {{ props.row.Background }}</p>
+
+          <p><b>Author:</b></p>
+          <p>{{ props.row.Author.join(', ') }}</p>
+
+          <p><b>Description:</b></p>
+          <p>{{ props.row.Description }}</p>
+
+          <p><b>Comments:</b></p>
+          <p>{{ props.row.Comments.join(', ') }}</p>
```
Comment (vinnybod) [inline src/views/Modules.vue]:
```suggestion
          <p>{{ props.row.Comments.join('\n ') }}</p>
```

**Comment by vinnybod** [comment]
Thanks! This will go into the next release

**Pull request merged**